lib/msf/core/db_manager/db_export.rb

```diff
@@ -0,0 +1,20 @@
+require 'msf/core/db_export'
+
+module Msf::DBManager::DbExport
+  def run_db_export(path, format)
+    exporter = Msf::DBManager::Export.new(framework.db.workspace)
+
+    output_file = exporter.send("to_#{format}_file".intern, path) do |mtype, mstatus, mname|
+      if mtype == :status
```

Comment:
Thoughts on pulled up the nested if statements and combining conditions to avoid nesting given there is no else logic on the outer if statement?

Reply:
Disclaimer: This was copy/pasted from ComandDispatcher.

But, I think having it nested makes more sense since it's only 1 check, then. If I moved it to be included in the other two `if` statements it would have two.